src/service.py

```diff
@@ -296,6 +297,9 @@ def service_enable(names):
         names = [names]
     for name in names:
         if _run_service_command("enable", name):
+            services = _get_services()
+            if name in services:
+                diagnosis_unignore({"services": [{"service": name}]})
```

Comment:
Eh apparently it will raise an exception if the filter does not exists ... https://github.com/YunoHost/yunohost/blob/6ed167bfafc4c4f27fa463790fcfd76960b96564/src/diagnosis.py#L325 maybe we should tweak the original code because it feels unecessary to raise an exception in that case. The "This filter already exists." one only shows a warning

Reply:
done ^w^
[`20741c6` (#1886)](https://github.com/YunoHost/yunohost/pull/1886/commits/20741c63aafe11f4443319a35dbfb4b00732c6ca)